screensharing/main.js

```diff
@@ -0,0 +1,113 @@
+const { app, BrowserWindow, ipcMain, screen } = require('electron');
+
+const { SCREEN_SHARE_EVENTS_CHANNEL, SCREEN_SHARE_EVENTS, TRACKER_SIZE } = require('./constants');
+
+/**
+ * Main process component that sets up electron specific screen sharing functionality, like screen sharing
+ * tracker and window selection.
+ * The class will process events from {@link ScreenShareRenderHook} initialized in the renderer, and the
+ * always on top screen sharing tracker window.
+ */
+class ScreenShareMainHook {
+    /**
+     * Create ScreenShareMainHook linked to jitsiMeetWindow.
+     *
+     * @param {BrowserWindow} jitsiMeetWindow - BrowserWindow where jitsi-meet api is loaded.
+     * @param {string} identity - Name of the application doing screen sharing, will be displayed in the
+     * screen sharing tracker window text i.e. {identity} is sharing your screen.
+     */
+    constructor(jitsiMeetWindow, identity) {
```

Comment:
I'm wondering if can get the `identity` value from electron directly. 

Is `app.name` https://www.electronjs.org/docs/api/app#appname going to do the job?

Reply:
You could but I prefer for it to be passed down as a parameter, whoever uses this can pass app.name if that's what they need. You would avoid cases where the app.name is very long and maybe not appropriate for the tracker window i.e. (8x8 Free Video Meetings or some other long name).

Reply:
OK. That's fine!